tests/test_api.py

```diff
@@ -467,3 +471,132 @@ def test_none_or_empty_data(self):
 
         with self.assertRaisesRegex(InvalidValueError, IDENTITY_NONE_OR_EMPTY_ERROR):
             api.add_identity('scm', name='', email='', username='')
+
+
+class TestDeleteIdentity(TestCase):
+    """Unit tests for delete_identity"""
+
+    def setUp(self):
+        """Load initial dataset"""
+
+        # Organizations
+        example_org = db.add_organization('Example')
+        bitergia_org = db.add_organization('Bitergia')
+        libresoft_org = db.add_organization('LibreSoft')
+
+        # Identities
+        jsmith = api.add_identity('scm', email='jsmith@example')
+        api.add_identity('scm',
+                         name='John Smith',
+                         email='jsmith@example',
+                         uuid=jsmith.id)
+        Enrollment.objects.create(uidentity=jsmith.uidentity,
+                                  organization=example_org)
+        Enrollment.objects.create(uidentity=jsmith.uidentity,
+                                  organization=bitergia_org)
+
+        jdoe = api.add_identity('scm', email='jdoe@example')
+        Enrollment.objects.create(uidentity=jdoe.uidentity,
+                                  organization=example_org)
+
+        jrae = api.add_identity('scm',
+                                name='Jane Rae',
+                                email='jrae@example')
+        Enrollment.objects.create(uidentity=jrae.uidentity,
+                                  organization=libresoft_org)
+
+    def test_delete_identity(self):
+        """Check whether it deletes an identity"""
+
```

Comment:
I would add some code to show the number of initiali unique identities and identities. SOmething like:
```
uidentities = UniqueIdentity.objects.all()
 self.assertEqual(len(uidentities), 4)

identities = Identity.objects.all()
self.assertEqual(len(identities), 4)
```